diff -uNr cphone-0.3.1/Makefile cphone-0.3.1-new/Makefile
--- cphone-0.3.1/Makefile	2003-08-07 00:39:42.000000000 +0000
+++ cphone-0.3.1-new/Makefile	2004-01-29 16:35:50.000000000 +0000
@@ -117,7 +117,7 @@
 UIC             = $(QTDIR)/bin/uic
 MOC             = $(QTDIR)/bin/moc
 
-STDCCFLAGS      += -I$(QTDIR)/include -DQT_THREAD_SUPPORT
+STDCCFLAGS      += -I$(QTDIR)/include/qt -DQT_THREAD_SUPPORT
 LDFLAGS         += -L$(QTLIBDIR) -L$(XLIBDIR)
 ENDLDLIBS       += -lqt-mt -lSM -lICE -lXext -lX11 
 ifeq ($(OSTYPE),linux)
@@ -141,7 +141,7 @@
 OPENH323DIR=$(HOME)/openh323
 endif
 
-include $(OPENH323DIR)/openh323u.mak
+include $(OPENH323DIR)/share/openh323/openh323u.mak
 
 ##################################################
 
